model/shared/src/main/scala/explore/data/KeyedIndexedList.scala

```diff
@@ -139,7 +145,7 @@ object KeyedIndexedList:
               .map(list =>
                 KeyedIndexedList.unsafeFromTreeSeqMap(
                   TreeSeqMap.from(
-                    list.sortBy { case (_, (_, i)) => i }
+                    list.sortBy { case (_, (_, i)) => i }(Order[NonNegInt].toOrdering)
```

Comment:
I'm not sure why, but I wasn't able to derive an `Ordering[NonNegInt]` here, so instead I summon the `Order[NonNegInt]` and convert that. If I'm missing an import then I'd be happy to replace it

Reply:
Usually, if you `import cats.Order.given`, it derives an `Ordering` from an implicit `Order` in scope. Not sure whether we can bring the `Order` given into scope here though.

Reply:
Yup, that fixed it